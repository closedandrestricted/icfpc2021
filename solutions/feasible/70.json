{"vertices":[[10,43],[16,40],[7,42],[13,45],[10,43],[14,38],[9,42],[6,48],[7,42],[12,44],[16,39],[12,44],[15,38],[7,42],[13,45],[5,45],[23,37],[18,41],[12,44],[6,48],[11,43],[17,40],[12,44],[5,45],[13,46],[15,39],[15,37],[9,41],[4,46],[13,46],[6,48],[8,44],[11,43],[6,48],[11,44],[15,39],[7,42],[1,45],[5,47],[9,42],[4,46],[12,45],[6,48],[11,45],[16,40],[10,42],[16,39],[9,42],[4,46],[21,34],[16,38],[11,44],[6,48],[10,41],[10,42],[5,47],[10,42],[16,38]]}