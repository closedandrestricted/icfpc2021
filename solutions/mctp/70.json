{"vertices":[[20,35],[19,28],[29,28],[23,31],[20,35],[15,40],[18,21],[22,26],[20,38],[21,34],[15,30],[16,34],[14,27],[18,32],[13,37],[24,40],[15,37],[22,37],[21,28],[25,34],[28,35],[21,33],[28,40],[24,35],[32,34],[38,37],[32,28],[26,26],[25,33],[32,35],[38,38],[46,34],[34,35],[27,33],[32,33],[39,33],[44,35],[41,29],[39,31],[33,29],[39,33],[46,34],[49,40],[33,29],[40,31],[46,31],[53,33],[54,44],[60,41],[37,25],[44,25],[51,40],[58,38],[67,39],[43,32],[50,33],[65,37],[71,33]]}