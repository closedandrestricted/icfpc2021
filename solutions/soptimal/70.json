{"vertices":[[12,24],[8,30],[6,42],[0,45],[22,15],[18,21],[14,33],[12,40],[6,48],[26,3],[27,10],[24,24],[21,30],[19,42],[13,46],[32,0],[33,15],[31,22],[27,34],[26,41],[37,4],[39,11],[36,27],[34,34],[32,43],[32,50],[44,1],[44,18],[44,25],[40,39],[39,46],[37,56],[48,7],[50,14],[49,30],[46,36],[46,46],[41,50],[55,7],[55,20],[55,27],[53,38],[52,45],[61,8],[61,15],[62,29],[60,36],[59,47],[65,51],[65,21],[68,27],[67,39],[66,46],[72,48],[74,28],[73,35],[72,48],[79,47]]}