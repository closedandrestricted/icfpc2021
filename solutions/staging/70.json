{"vertices":[[26,11],[22,16],[15,34],[12,40],[31,14],[25,17],[19,23],[14,28],[6,45],[38,7],[38,14],[27,23],[23,29],[15,35],[13,42],[42,12],[33,19],[26,17],[18,34],[18,41],[43,18],[39,23],[24,24],[24,31],[25,41],[29,35],[48,24],[34,16],[31,22],[31,34],[32,41],[32,43],[48,17],[41,15],[37,23],[37,30],[38,42],[37,49],[50,10],[48,17],[44,23],[43,35],[45,42],[57,8],[55,15],[51,24],[49,31],[51,44],[58,46],[62,18],[58,24],[54,37],[58,42],[64,41],[64,27],[58,31],[64,46],[57,44]]}