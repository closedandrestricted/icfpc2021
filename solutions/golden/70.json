{"vertices":[[64,20],[67,26],[67,39],[66,46],[61,16],[58,22],[69,33],[62,35],[59,47],[61,9],[55,13],[55,28],[62,30],[56,39],[52,45],[55,7],[52,20],[49,26],[55,32],[50,36],[48,9],[46,16],[42,27],[48,31],[51,43],[45,46],[44,2],[38,14],[37,21],[43,38],[43,45],[39,52],[42,8],[35,7],[34,27],[38,32],[36,47],[43,46],[36,4],[34,14],[36,21],[31,34],[32,41],[29,4],[27,11],[29,22],[26,28],[29,35],[25,41],[22,17],[22,24],[20,33],[23,39],[18,42],[15,23],[13,30],[16,40],[11,45]]}